tests/integration/builtin_functions/resources/taste/builtin_functions.asn

```diff
@@ -22,5 +22,7 @@ BEGIN
        charge MyInteger OPTIONAL
    }
 
+   MyFixed ::= SEQUENCE (SIZE(2)) OF MyInteger
+
```

Comment:
Just to be safe, and have this topic closed, let's have a yet another tested type - MyDerivedFixed ::= MyFixed and MyDerivedArray ::= MyArray. What do you think?